### ISSUE TITLE
Border on the bottom&right of generated pdf

### ISSUE THREAD

**Issue by devnes** [wkhtmltopdf/wkhtmltopdf#3154]
Hi there,
I have setup no borders for pdf layout (full page).
But if I zoom my PDF, I have some borders on the bottom and right side.

![](http://i.shotnes.com/u/1/1006/screenshot_24756.png)

> wkhtmltopdf --margin-top 0 --margin-bottom 0 --margin-left 0 --margin-right 0 --exclude-from-outline --no-outline --dpi 300 --disable-smart-shrinking

> *{margin:0;padding:0;}

How remove this borders?

**Comment by ajoah** [comment]
Same problem :/ Did you find a solution ?

**Comment by devnes** [comment]
Yes, after many attempts, I found a solution, set --page-height pdf_size+0.1, for example: --page-height 300.1mm

**Comment by ajoah** [comment]
Thank you for your answer but no effect in my case :(

**Comment by devnes** [comment]
I create file wkhtmltopdf.sh:
`xvfb-run -a -s "-screen 0 1920x1080x24" /usr/bin/wkhtmltopdf --margin-top 0 --margin-bottom 0 --margin-left 0 --margin-right 0 --exclude-from-outline --no-outline --dpi 300 --disable-smart-shrinking "$@"
`
and in php:
`passthru('wkhtmltopdf.sh --page-width '.($pdf_size).'mm --page-height '.($pdf_size+0.1).'mm http://'.$url.' '.$file);`

And I think it's important, I use wkhtmltopdf based on Qt (0.12.4 (with patched qt)).
I hope this will help you.

**Comment by devnes** [comment]
Unfortunately yes. 

**html example:**
```html
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<style>
    @page { size: 210mm 155mm }
    *{margin:0;padding:0;}
    body { margin: 0; padding: 0; }
   .item {
        width: 210mm;
        height: 155mm;
   }
</style>

<div class="item" style="background-color: #f38f8f;"></div>
<div class="item" style="background-color: #6f67ec;"></div>
<div class="item" style="background-color: green;"></div>
<div class="item" style="background-color: #e4ec67;"></div>
```

**I use command:** 
`wkhtmltopdf --margin-top 0 --margin-bottom 0 --margin-left 0 --margin-right 0 --exclude-from-outline --no-outline --dpi 300 --disable-smart-shrinking --page-width 210mm --page-height 155mm my_html.html res.pdf`

result you can see here [res.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/2446629/res.pdf)  you need to zoom the bottom of the page, and can see how on every next page the previous one creeps more and more.
![image](https://user-images.githubusercontent.com/5395316/46479797-8f9c4480-c7f8-11e8-99c9-7b90c61a8b43.png)
